components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/authz/OAuthAuthzReqMessageContextTest.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.identity.oauth2.authz;
+
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.wso2.carbon.identity.oauth2.dto.OAuth2AuthorizeReqDTO;
+import static org.testng.Assert.assertEquals;
+import java.lang.String;
+
+public class OAuthAuthzReqMessageContextTest {
+    OAuth2AuthorizeReqDTO authorizationReqDTON = new OAuth2AuthorizeReqDTO();
+
+    private OAuthAuthzReqMessageContext oauthAuthzReqMessageContext;
+    private OAuth2AuthorizeReqDTO authorizationReqDTO;
+
+    @BeforeTest
+    public void setUp() {
+        OAuth2AuthorizeReqDTO authorizationReqDTON = new OAuth2AuthorizeReqDTO();
+        oauthAuthzReqMessageContext = new OAuthAuthzReqMessageContext(authorizationReqDTON);
+        this.authorizationReqDTO = authorizationReqDTON;
+    }
+
+    @Test
+    public void testSetAuthorizationReqDTO() throws Exception {
+        OAuth2AuthorizeReqDTO authorizationReqDTON = new OAuth2AuthorizeReqDTO();
+        authorizationReqDTON.setEssentialClaims("https://www.wso2.org/name");
+        assertEquals(authorizationReqDTON.getEssentialClaims(),
+                "https://www.wso2.org/name" , "Valid claim value.");
+
+        authorizationReqDTON.setConsumerKey("0289");
+        assertEquals(authorizationReqDTON.getConsumerKey(),
+                "0289" , "Valid consumer key.");
+    }
+
+    @Test
+    public void testGetApprovedScope() throws Exception {
```

Comment:
We can test all getter/setters in single method. No need to have separate method per each setter and getter.

Reply:
Fixed this by removing extra getter methods